save-backend/src/main/kotlin/org/cqfn/save/backend/postprocessor/DockerSecretsDatabaseProcessor.kt

```diff
@@ -14,7 +14,7 @@ import java.util.Properties
 /**
  * Post processor that collects credentials for database from docker secrets
  */
-@Profile("prod")
+@Profile("docker-secrets")
```

Comment:
Currently we use a custom postprocessor to read database credentials, but in Kubernetes it will be done internally by spring-cloud-kubernetes and consumed as an ordinary PropertySource. So to keep compatibility with pure Docker deployment I'm moving this component into a separate profile